reference/import_export/formats/scopable-labels.rst

```diff
@@ -0,0 +1,15 @@
+Scopable labels
+===============
+
+In the case of product export, all localized and/or scopable labels follow the same rule in the import/export:
+
+{code}-{scope}-{locale}
+
+Thus, a `foo` label available in ecommerce and print channels and in fr_FR, en_US and de_DE locales
+will be exported in six columns:
```

Comment:
And what if my attribute is just scopable and not localizable ? You should explain it so people don't think scopable = localizable.